Templates/MonoGame.Templates.CSharp/content/MonoGame.2D.StartKit/___SafeGameName___.Core/Content/Fonts/Hud.spritefont

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+This file contains an xml description of a font, and will be read by the XNA
+Framework Content Pipeline. Follow the comments to customize the appearance
+of the font in your game, and to change the characters which are available to draw
+with.
+-->
+<XnaContent xmlns:Graphics="Microsoft.Xna.Framework.Content.Pipeline.Graphics">
+  <Asset Type="Graphics:FontDescription">
+
+    <!--
+    Modify this string to change the font that will be imported.
+    -->
+    <FontName>Tahoma</FontName>
```

Comment:
Is this font available on all Desktop Platforms? If not can we get one that is or ship one with the template? (We would need rights to be able to ship a font).

Reply:
This is placeholder for now, for testing in Windows. But we definitely need one we can ship.

Reply:
@dellis1972 @SimonDarksideJ I found this dude https://fonty.condak.cz/index.php who has quite a few completely FREE fonts, as well as commercial ones. Do you think it is worth us hosting the free ones, in case he disappears, and we store/promote them (with attribution of course) as being part of MonoGame and free for devs to you?